### ISSUE TITLE
Hello, may I ask if you could open-source the code for processing datasets?

### ISSUE THREAD

**Issue by zkyoma** [xbh0720/LBKT#1]

**Comment by xbh0720** [comment]
You can refer to this link https://github.com/bigdata-ustc/EduKTM/blob/main/examples/LBKT/data_preprocess.ipynb where I provide an example of processing ASSISTMENT2019. I will synchronize the code as soon as possible.

**Comment by zkyoma** [comment]
> You can refer to this link https://github.com/bigdata-ustc/EduKTM/blob/main/examples/LBKT/data_preprocess.ipynb where I provide an example of processing ASSISTMENT2019. I will synchronize the code as soon as possible.

thank you!

**Comment by zkyoma** [comment]
> You can refer to this link https://github.com/bigdata-ustc/EduKTM/blob/main/examples/LBKT/data_preprocess.ipynb where I provide an example of processing ASSISTMENT2019. I will synchronize the code as soon as possible.

I used the PyTorch version of the code from the repository link, and the results on the 2009 dataset are quite different from the ones in the paper. I only adjusted the batch_size, and the other hyperparameters remained unchanged. Can you help me understand the reason behind this difference?

auc: 0.724535, accuracy: 0.672008, rmse: 0.446850
